src/Dash.NET/DashComponents/CoreComponents/Input.fs

```diff
@@ -90,190 +184,510 @@ module Input =
+        ///</summary>
+        static member children(value: float) = Children([ Html.Html.text value ])
+        ///<summary>
+        ///The child or children of this dash component
+        ///</summary>
+        static member children(value: System.Guid) = Children([ Html.Html.text value ])
+        ///<summary>
+        ///The child or children of this dash component
+        ///</summary>
+        static member children(value: DashComponent) = Children([ value ])
+        ///<summary>
+        ///The child or children of this dash component
+        ///</summary>
+        static member children(value: list<DashComponent>) = Children(value)
+        ///<summary>
+        ///The child or children of this dash component
+        ///</summary>
+        static member children(value: seq<DashComponent>) = Children(List.ofSeq value)
+
+    ///<summary>
+    ///A basic HTML input control for entering text, numbers, or passwords.
+    ///Note that checkbox and radio types are supported through
+    ///the Checklist and RadioItems component. Dates, times, and file uploads
+    ///are also supported through separate components.
+    ///</summary>
     type Input() =
         inherit DashComponent()
         static member applyMembers
             (
-                id : string,
-                children : seq<DashComponent>,
-                ?Value               : IConvertible,
-                ?Style               : DashComponentStyle ,
-                ?ClassName           : string,
-                ?Debounce            : bool,
-                ?Type                : InputType,
-                ?AutoComplete        : string,
-                ?AutoFocus           : bool,
-                ?Disabled            : bool,
-                ?Mode                : InputMode,
-                ?List                : string,
-                ?Max                 : IConvertible,
-                ?MaxLength           : IConvertible,
-                ?Min                 : IConvertible,
-                ?MinLength           : IConvertible,
-                ?Multiple            : bool,
-                ?Name                : string,
-                ?Pattern             : string,
-                ?Placeholder         : IConvertible,
-                ?ReadOnly            : bool,
-                ?Required            : bool,
-                ?SelectionDirection  : string,
-                ?SelectionEnd        : string,
-                ?SelectionStart      : string,
-                ?Size                : string,
-                ?Spellcheck          : SpellCheckOptions,
-                ?Step                : IConvertible,
-                ?NSubmit             : IConvertible,
-                ?NSubmitTimestamp    : IConvertible,
-                ?NBlur               : IConvertible,
-                ?NBlurTimestamp      : IConvertible,
-                ?SetProps            : obj,
-                ?LoadingState        : LoadingState,
```

Comment:
Is the component generator able to create DUs from enumerated strings? Because this is one of the main advantages i can see for a language such as F# bringing to the table for these cases. I see that all the arguments that were wrapped like this seem to loose the DUs currently.

Reply:
Yes it can. I removed them here mostly because the goal of this PR was to convert the existing core components to feliz style and wanted to avoid changing too much else all at once, and to cut down on testing since we already know that these types work.

Would it make sense for me to add them in in this PR?

Reply:
Ah, I see what you mean now. I had forgotten to use the type-safe types with `init` and `applyMembers`. 

Though looking further, do we even need `init` and `applyMembers` (at least for anything other than `id` and `children`)? I added them very early into component generation to replicate what existed, but evidently forgot about them because they were barely used.

Reply:
For now I have fixed the type safety in relation to the existing types in `Props` on `init` and `applyMembers` (as well as regenerated the commented out components after fixing some component generation bugs).

The types themselves however still match the old versions of the components, as I am unsure if I should update them to match the generated code yet.

Reply:
> Though looking further, do we even need init and applyMembers (at least for anything other than id and children)? I added them very early into component generation to replicate what existed, but evidently forgot about them because they were barely used.

Thats another API layer basically. Arguably we do not need it in F# as we have the nice React DSLs, but i am pretty sure that is not as easy to use from C#. As we use `DynamicObj` anyways, we might as well add an option to initialize the component with a more `constructor style` (which is `init`). 

For F# we have the benefit of adding a more declarative way of creating and also styling existing components via `applyMembers` as well for those who do not like any kind of React DSL.

The pattern is coming from Plotly.NET where we use it with great success, and as we can generate components now it also has no added cost in terms of writing code anymore